msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -955,14 +955,15 @@ The `extensions` property specifies Outlook Add-ins within an app manifest and s
 |`autoRunEvents`| Array | | | Defines the event-based activation extension point. |
 |`alternates`| Array | | | Specifies the relationship to alternate existing Microsoft 365 solutions. It's used to hide or prioritize add-ins from the same publisher with overlapping functionality. |
 |`audienceClaimUrl`| String | 2048 characters | | Specifies the URL for your extension and is used to validate Exchange user identity tokens. For more information, see [inside the Exchange identity token](/office/dev/add-ins/outlook/inside-the-identity-token)|
+|`contextMenus`| Array | | | Specifies the context menus for your extension. A context menu is a shortcut menu that appears when you right-click (or select and hold) in an open Office document. |
 
 For more information, see [Office Add-ins manifest for Microsoft 365](/office/dev/add-ins/develop/unified-manifest-overview).
```

Comment:
Hi @ElizabethSamuel-MSFT ,
Observed that the feature document referred here does not contain 'contextMenus' under the section "extensions" property and in the sample unified manifest. Please do the needful.

Reply:
@v-ramyas5 
The unified-manifest-overview article is intended to be an overview, not a replacement for the reference documentation of the “extensions” property. I will update it to make it clear that the table shows only certain selected and representative child properties, not all of them.

We should not update the sample manifest. It is not possible to have a sample with every possible property anyway because some are mutually exclusive. Note that the sample manifests in the overview for the add-in only manifest also don’t have every possible element.

Reply:
Hey @Rick-Kirkham,
Sorry for the delayed response. I understand that the doc may not have all the fields. It is nice-to-add context menu to help new developers understand the feature better. Just a small suggestion from our side :)